discirc/irc_bot.py

```diff
@@ -41,6 +42,7 @@ class IRCBot(bottom.Client):
 
     COLOR_PREF = '\x03'
     CANCEL = '\u000F'
+    IRC_MSG_CHUNK_LENGTH = 256 # chunk length in letters - actual limit is 512 bytes, we anticipate some Unicode characters in the content
```

Comment:
Not PEP8 compliant : 
You need 2 spaces on an inline comment and your line is too long please consider 

```python
# chunck length in letters - actual limit is 512 bytes
# set to 256 to anticipate some Unicode chars in the content
IRC_MSG_CHUNK_LENGTH = 256
```